civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -562,16 +562,47 @@ function civihr_employee_portal_init() {
       '/lib/sweetalert/sweetalert.min.js'
     ]);
 
+    civicrm_resources_load('org.civicrm.reqangular', ['reqangular.min.js']);
+    _add_script_to_page('/js/ta-documents-app.js', 'dashboard');
+
     _rebuild_view('absence_list');
     _rebuild_view('length_of_service');
     _rebuild_view('absence_activity');
     _rebuild_view('hrvacancy');
     _rebuild_view('appraisal');
     _rebuild_view('tasks');
     _rebuild_view('documents');
+
+    _load_ta_settings();
   }
 }
 
+/**
+ * Pass tasksAssignments data to Global Drupal.settings object
+ */
+function _load_ta_settings() {
+  $config = CRM_Core_Config::singleton();
+  $taSettings = civicrm_api3('TASettings', 'get')['values'];
+
+  if (!empty($taSettings['days_to_create_a_document_clone'])) {
+    $taSettings['days_to_create_a_document_clone'] = (int)$taSettings['days_to_create_a_document_clone'];
+  }
+
+  drupal_add_js([
+    'tasksAssignments' => [
+      'extensionPath' => CRM_Core_Resources::singleton()->getUrl('uk.co.compucorp.civicrm.tasksassignments'),
+      'case_extension' => !empty(CRM_Core_Component::get('CiviCase')),
+      'settings' => $taSettings,
+      'permissions' => [
+        'delete_tasks_and_documents' => CRM_Core_Permission::check('delete Tasks and Documents'),
+      ],
+      'debug' => $config->debug
+    ],
+    'adminId' => CRM_Core_Session::getLoggedInContactID(),
+    'contactId' => CRM_Utils_Request::retrieve('cid', 'Integer')
```

Comment:
Can you explain to me what you need `adminId` and `contactId` for?  I saw your explanations for both of them but it still doesn't make much sense to me